Tag processor

asgerb opened the pull request:
Here we go finally :)

I’ve updated the readme too and removed nokogiri, which was included as a development dependency by mistake.

Oh and I didn’t touch the gem’s version, am guessing this is better to do in the master branch.

Commit:
Init

tomasc committed:
update

Commit:
Remove nokogiri

Don’t think we need this, seems to have been included by mistake when
first creating the gem?

Commit:
Update readme

tomasc committed:
Update .travis.yml

tomasc commented on lib/dragonfly_audio/plugin.rb:
```diff
@@ -1,56 +1,31 @@
-
     def call app, opts={}
       app.add_analyser :audio_properties, DragonflyAudio::Analysers::AudioProperties.new
 
```
@asgerb wouldnt it make sense to use the AudioProprties class constants here so we dont list the same props in two places?

tomasc commented on lib/dragonfly_audio/processors/tag.rb:
```diff
@@ -0,0 +1,29 @@
+
+module DragonflyAudio
+  module Processors
+    class Tag
```
I guess same here, would it be possible to use the constants from AudioProperties classs?

tomasc commented on test/dragonfly_audio/plugin_test.rb:
```diff
@@ -12,6 +12,30 @@ module DragonflyAudio
         audio.must_respond_to :audio_properties
       end
+
+      %i(
```
Here as well, lets reuse the constants.

tomasc reviewed (commented):
A few ideas for improvements

tomasc committed:
Update README.md

Commit:
Use DragonflyAudio::Analysers::AudioProperties constants

Keeping the constant in the Tag processor, but pointing it to the
AudioProperties constant for clarity, makes sense?
